NotesApp/Scenes/Editor/EditorViewController.swift

```diff
@@ -0,0 +1,206 @@
+    private lazy var titleTextView: UITextView = {
+        let textView = UITextView()
+        textView.font = UIFont.systemFont(ofSize: 24, weight: .bold)
+        textView.textColor = .white
+        textView.backgroundColor = .clear
+        textView.delegate = self
+        textView.isEditable = true
+        textView.isScrollEnabled = false
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        return textView
+    }()
+
+    private lazy var bodyTextView: UITextView = {
+        let textView = UITextView()
+        textView.font = UIFont.systemFont(ofSize: 18)
+        textView.textColor = .white
+        textView.backgroundColor = .clear
+        textView.delegate = self
+        textView.isEditable = true
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        return textView
+    }()
+
+    // MARK: - Lifecycle
+    init(viewModel: EditorViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        setupBindings()
+        viewModel.loadData()
+        saveEmptyNoteIfNeeded()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.setNavigationBarHidden(true, animated: animated)
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        navigationController?.setNavigationBarHidden(false, animated: animated)
+    }
+
+    // MARK: - UI Setup
+    private func setupUI() {
+        view.backgroundColor = UIColor(hex: "#252525")
+        
+        setupCustomBackButton()
+        
+        titleTextView.addDoneButtonToolbar()
+        bodyTextView.addDoneButtonToolbar(onDone: (target: self, action: #selector(customDoneAction)))
+        
+        view.addSubview(titleTextView)
+        view.addSubview(bodyTextView)
+        
+        titleTextView.translatesAutoresizingMaskIntoConstraints = false
+        bodyTextView.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate([
+            titleTextView.topAnchor.constraint(equalTo: customBackButton.bottomAnchor, constant: 20),
+            titleTextView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            titleTextView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            
+            bodyTextView.topAnchor.constraint(equalTo: titleTextView.bottomAnchor, constant: 10),
+            bodyTextView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            bodyTextView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            bodyTextView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: 20)
+        ])
+    }
```

Comment:
As already mentioned, create seperate func or constraints and for hierarchy. you can remain only ui part in this func